### ISSUE TITLE
twirp-ts generates wrong import path for nested protos

### ISSUE THREAD

**Issue by tobiaskohlbau** [hopin-team/twirp-ts#11]
If a codebase contains it's protobuf files in nested directories the resulting twirp code tries to import via the _relative_ path based on the protoc call. The resulting import path is not valid and breaks the generated code.

Based on the example included in this repository I've created a repo which shows/reproduces this error at https://github.com/tobiaskohlbau/twirp_ts_reproducer. The _broken_ import is generated in the [service.twirp.ts](https://github.com/tobiaskohlbau/twirp_ts_reproducer/blob/fd7206ff3a081616a5258c36e06a53d7228dc478/generated/api/v1/service.twirp.ts#L11) file:

```
import { Size, Hat, FindHatRPC, ListHatRPC } from "./api/v1/service";
```

this should IMO be:
```
import { Size, Hat, FindHatRPC, ListHatRPC } from "./service";
```

I've traced down the _issue_ to the following line:
https://github.com/hopin-team/twirp-ts/blob/b4abf8c2f24172203f7ac070645e36b9b7e64dfc/src/protoc-gen-twirp-ts/gen/twirp.ts#L405
where the import is based on the _filename_ from the file. Protobuf states that the plugin infrastructure always uses the relative to selected path as filename.

```
    // The file name, relative to the output directory.  The name must not
    // contain "." or ".." components and must be relative, not be absolute (so,
    // the file cannot lie outside the output directory).  "/" must be used as
    // the path separator, not "\".
    //
    // If the name is omitted, the content will be appended to the previous
    // file.  This allows the generator to break large files into small chunks,
    // and allows the generated text to be streamed back to protoc so that large
    // files need not reside completely in memory at one time.  Note that as of
    // this writing protoc does not optimize for this -- it will read the entire
    // CodeGeneratorResponse before writing files to disk.
    optional string name = 1;
```

https://github.com/protocolbuffers/protobuf/blob/4b77b733699f9c39b517a124da737c018b1164e2/src/google/protobuf/compiler/plugin.proto#L133

One possible option would be to always stick to the relative `./{NAME}` import and strip away all other directories. But I'm not sure if this covers all cases.

**Comment by fenos** [comment]
Hi, I'm back from holiday!

Will get this fixed ASAP

**Comment by fenos** [comment]
@tobiaskohlbau  this is now fixed in `v2.1.2` Thanks for reporting

**Comment by tobiaskohlbau** [comment]
Awesome thanks for the fast fix, much appreciate it!
